code/AdobeEdge.brs

```diff
@@ -762,7 +762,7 @@ function _adb_serviceProvider() as object
                     ' else
                     '     print message
                     ' end if
-                    print "[" + _adb_ISO8601_timestamp() + "]" + message
+                    print "[" + _adb_ISO8601_timestamp().Replace("T", " ") + "]" + message
```

Comment:
We should skip this change. Don't want to introduce replace, considering the number of times this will be called. We could address this by adding a new util to extract date and time and then create a string out of it.

Reply:
OK, let me close this PR for now.